dockertest/subtestbase_unittests.py

```diff
@@ -48,44 +48,31 @@ class DockerTestFail(Exception):
 # END   boilerplate crap needed for subtests, which should be refactored
```

Comment:
I did, and they all passed...unless maybe I screwed something up after and forgot to run them again.